### ISSUE TITLE
Hyperleukocytosis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10782]
**Preferred term label:**
Hyperleukocytosis

**Synonyms**
Leukocyte count on complete blood count >100,000/uL.

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0001974

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@adamklocperk Thoughts on this concept? It would be a child or sib of [Increased total leukocyte count HP:0001974](https://hpo.jax.org/browse/term/HP:0001974)
I would define as follows
Def: Hyperleukocytosis is commonly defined as a white blood cell (WBC) count of greater than 100,000 luekocytes per microliter.
Comment: Hyperleukocytosis can be observed in conditions such as acute myeloid leukemia, which is characterized by the clonal proliferation of hematopoietic stem cells. Hyperleukocytosis is regarded as a hematological emergeny and can lead to complications such as leukostasis, disseminated intravascular coagulation, and tumor lysis syndrome.

**Comment by adamklocperk** [comment]
@pnrobinson I think that makes sense :)

**Comment by MickeySegal** [comment]
For a few findings we have terms for super-high levels, e.g., https://hpo.jax.org/browse/term/HP:0012593

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001425
- label: Hyperleukocytosis
- parents: [Increased total leukocyte count (HP:0001974)](https://hpo.jax.org/app/browse/term/HP:0001974)
- pmid: PMID:33080779
- def: Hyperleukocytosis is commonly defined as a white blood cell (WBC) count of greater than 100,000 luekocytes per microliter.
- comment: Hyperleukocytosis can be observed in conditions such as acute myeloid leukemia, which is characterized by the clonal proliferation of hematopoietic stem cells. Hyperleukocytosis is regarded as a hematological emergeny and can lead to complications such as leukostasis, disseminated intravascular coagulation, and tumor lysis syndrome.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10782
- synonyms:
